pkg/apis/pravega/v1alpha1/pravega.go

```diff
@@ -129,10 +137,39 @@ func (s *PravegaSpec) withDefaults() (changed bool) {
 		changed = true
 		s.Tier2 = &Tier2Spec{}
 	}
+
 	if s.Tier2.withDefaults() {
 		changed = true
 	}
 
+	if s.ControllerResources == nil {
+		changed = true
+		s.ControllerResources = &v1.ResourceRequirements{
+			Requests: v1.ResourceList{
+				v1.ResourceCPU:    resource.MustParse("250m"),
```

Comment:
Same comment here with the defaults compared to `cr-detailed.yaml`.

Reply:
Responded in https://github.com/pravega/pravega-operator/pull/129#discussion_r267449637